HW02_A_Graph-Fails.Rmd

```diff
@@ -82,6 +146,17 @@ ggplot(data = mpg, mapping = aes(x = manufacturer, y = cty, color = manufacturer
   geom_boxplot() + 
   theme(axis.text.x = element_text(angle = 45))
 ```
+#CORRECTED CODE for Graph 5
+To make the box one color instead of just the outline, use fill instead of color. 
+
+To adjust the x-axis labels such that they aren't on the graph, use margin. I initially tried using vjust such that my code read theme(axis.title.x = element_text(vjust = -.05)) but I saw no results. I then stumbled upon someone commenting on a stackoverflow post that since ggplot2, the vjust parameter has been superceded by margins and that is a better way to move text around. Just fyi, margins can be specified for top (t), right (r), bottom (b) and left (l) to position text vertically and horizontally. 
+

```

Comment:
You have the right idea using vjust ... I was able to accomplish this using hjust, my code was as follows:

theme(axis.text.x = element_text(angle = 45, hjust = 1))

hjust controls horizontal justification and vjust controls vertical justification so I think you may have just picked the wrong one.